src/ai/backend/manager/data/vfolder/dto.py

```diff
@@ -114,3 +116,14 @@ def to_field(self) -> VFolderItemField:
             max_files=self.max_files,
             cur_size=self.cur_size,
         )
+
+
+class ServiceEnvironmentConfig(BaseModel):
+    image: str
+    architecture: str
+
+
+class ServiceConfig(BaseModel):
+    runtime_variant: RuntimeVariant
+    environment: ServiceEnvironmentConfig
+    resource_limits: dict[str, Any] = Field(default_factory=dict)
```

Comment:
I guess there should be a separation between the service config that goes into the dto and the internal use service config, but how are you thinking of proceeding with the part that uses the service config?